test/e2e-go/features/compactcert/compactcert_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2019-2020 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package compactcert
+
+import (
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/algorand/go-algorand/config"
+	"github.com/algorand/go-algorand/crypto"
+	"github.com/algorand/go-algorand/crypto/compactcert"
+	"github.com/algorand/go-algorand/daemon/algod/api/spec/v1"
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/transactions"
+	"github.com/algorand/go-algorand/protocol"
+	"github.com/algorand/go-algorand/rpcs"
+	"github.com/algorand/go-algorand/test/framework/fixtures"
+)
+
+func TestCompactCerts(t *testing.T) {
+	t.Parallel()
+	r := require.New(t)
+
+	configurableConsensus := make(config.ConsensusProtocols)
+	consensusVersion := protocol.ConsensusVersion("test-fast-compactcert")
+	consensusParams := config.Consensus[protocol.ConsensusCurrentVersion]
+	consensusParams.CompactCertRounds = 8
+	consensusParams.CompactCertTopVoters = 1024
+	consensusParams.CompactCertVotersLookback = 2
+	consensusParams.CompactCertWeightThreshold = 30
+	consensusParams.CompactCertSecKQ = 128
+	configurableConsensus[consensusVersion] = consensusParams
+
+	var fixture fixtures.RestClientFixture
+	fixture.SetConsensus(configurableConsensus)
+	fixture.Setup(t, filepath.Join("nettemplates", "CompactCert.json"))
+	defer fixture.Shutdown()
+
+	restClient, err := fixture.NC.AlgodClient()
+	r.NoError(err)
+
+	var lastCertBlock v1.Block
+	libgoal := fixture.LibGoalClient
+	for rnd := uint64(1); rnd <= consensusParams.CompactCertRounds*4; rnd++ {
+		fixture.WaitForRound(rnd, 30*time.Second)
+		blk, err := libgoal.Block(rnd)
+		r.NoError(err)
+
+		// fmt.Printf("Round %d, block %v\n", rnd, blk)
```

Comment:
remove this?

Reply:
I changed it to `t.Logf()` instead, which captures what I actually wanted: it's a useful debug message if the test is failing, but doesn't need to be printed otherwise.